ci-operator/config/openshift/kubernetes-autoscaler/openshift-kubernetes-autoscaler-master.yaml

```diff
@@ -58,3 +58,13 @@ tests:
   commands: ARTIFACT_DIR=/tmp/artifacts JUNIT_REPORT=1 hack/test-go.sh
   container:
     from: src
+- as: e2e-aws
```

Comment:
The autoscaler is only deployed as a post install activity so via cluster autoscaler operator which will be tested in e2e-aws-operator e.g https://github.com/openshift/machine-api-operator/pull/177 so I wonder if there's any value on running `e2e-aws` cc @frobware @bison

Reply:
Yeah, that's fair. As things stand today, these tests couldn't possibly catch anything, right? We mainly want an e2e suite on AWS specific to the autoscaler.

Reply:
Fair enough. I will remove those